client/components/game-list.jsx

```diff
@@ -12,10 +10,37 @@ export default function GameList(props) {
               key={single.name}
               title={single.name}
               img={single.image.super_url}
+              id={single.id}
+              description={single.description}
+              deck={single.deck}
+              getGame={props.getGame}
             />
           );
         })
       }
     </ul>
   );
 }
+
+function SingleGame(props) {
+  const { id, title, img, deck, description } = props;
+  const bringBack = () => {
+    props.getGame(id, title, img, deck, description);
+  };
+  return (
+    <div className="row-game" onClick={bringBack}>
+      <div className="img-container">
+        <img src={img}></img>
```

Comment:
we should try forcing them to be langscape. it probably won't look as bad. You have to send the image height and width, but then use object-fit in the CSS. i think it will look WAY better event if some of the image is cut off